modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Activities/Activities.tsx

```diff
@@ -127,30 +123,8 @@ const Activities = ({
 		updateEditableActivity();
 	};
 
-	const onRemove = async (index: number) => {
-		if (values.activities[index].id) {
-			try {
-				if (values.activities[index].externalReferenceCodeSF) {
-					await deleteMDFRequestActivitiesSF(
-						ResourceName.ACTIVITY_SALESFORCE,
-						values.activities[index]
-							.externalReferenceCodeSF as string
-					);
-				}
-
-				await deleteMDFRequestActivities(
-					ResourceName.ACTIVITY_DXP,
-					values.activities[index].id as number
-				);
-			}
-			catch (error: any) {
-				handleError(error.message);
-
-				return;
-			}
-		}
-
-		arrayHelpers.remove(index);
+	const onRemove = (index: number) => {
+		setFieldValue(`activities[${index}].onRemove`, true);
```

Comment:
since it just a function called another, try to use the arrow function call.

```suggestion
	const onRemove = (index: number) => setFieldValue(`activities[${index}].removed`, true)
```